images/virtualization-artifact/pkg/controller/cvi/internal/source/registry.go

```diff
@@ -147,6 +153,13 @@ func (ds RegistryDataSource) Sync(ctx context.Context, cvi *virtv2.ClusterVirtua
 			}
 		}
 
+		ds.recorder.Event(
+			cvi,
+			corev1.EventTypeNormal,
+			virtv2.ReasonDataSourceSyncCompleted,
+			"The HTTP DataSource import has completed",
```

Comment:
copy paste error: it's not a http data source

Reply:
fixed